### ISSUE TITLE
Self-signed SSL certificates return errors

### ISSUE THREAD

**Issue by hassy** [artilleryio/artillery#1006]
hmm,  I tried both the flag and tls in the config, but I'm still getting this error for all requests... 🤔

```
  Errors:
    SELF_SIGNED_CERT_IN_CHAIN: 1
```

_Originally posted by @JimLynchCodes in https://github.com/artilleryio/artillery/issues/8#issuecomment-827642795_

**Comment by hassy** [comment]
also see: https://github.com/artilleryio/artillery/issues/8#issuecomment-827880143

**Comment by hassy** [comment]
@JimLynchCodes & @bluezombie - could you confirm if this is on Artillery v1.7 (`artillery -V`)?

**Comment by bluezombie** [comment]
> @JimLynchCodes & @bluezombie - could you confirm if this is on Artillery v1.7 (`artillery -V`)?

`------------ Version Info ------------
Artillery: 1.6.2
Artillery Pro: not installed (https://artillery.io/pro)
Node.js: v16.0.0
OS: linux/x64
--------------------------------------
`
@hassy I can confirmed that using the -k parameter worked perfectly. My problem was I used the "rejectUnauthorized" parameter in the yml file plus using the -k in the command line: this didn't work out. Then, I removed the "rejectUnauthorized" from the YML file, and use the -k parameter and worked.
